components/concept/engine/src/main/java/mozilla/components/concept/engine/manifest/WebAppManifestParser.kt

```diff
@@ -125,7 +125,7 @@ private fun parseIcons(json: JSONObject): List<WebAppManifest.Icon> {
             WebAppManifest.Icon(
                 src = obj.getString("src"),
                 sizes = parseIconSizes(obj),
-                type = obj.optString("type", null),
+                type = obj.tryGetString("type"),
```

Comment:
These `tryGetString` changes belong in an earlier PR no?

Land them here if you have already landed an early transformation.

Reply:
I already landed `tryGetString` previously in #3453, I just missed a few (or perhaps they slipped in).